Hw25.web server

BychkovVA opened the pull request:
+build gradle

BychkovVA committed:
hw25 web server

BychkovVA committed:
+bild gradle

idontgiveit commented on hw-25.web-server/src/main/java/atm/controller/FormsController.java:
```diff
@@ -0,0 +1,78 @@
+@Controller
+public class FormsController {
+
+    private static  ATM atm = new ATM();
```
У вас же спринг, Наверное хотелось бы чтобы сервисы инжектились, иначе зачем это все )

idontgiveit commented on hw-25.web-server/src/main/java/atm/controller/FormsController.java:
```diff
@@ -0,0 +1,78 @@
+        String message;
+
+        switch (action) {
+            case "takeMoney":
```
Я придираться не буду, но обычно это константами делают.

idontgiveit commented on hw-25.web-server/src/main/java/atm/service/ATM.java:
```diff
@@ -0,0 +1,69 @@
+    }
+
+
+    public String withdrawAmount(int amount) {
```
Вот здесь, кстати, нормально сделано.

idontgiveit commented on hw-25.web-server/src/main/java/atm/service/ATM.java:
```diff
@@ -0,0 +1,69 @@
+    }
+
+    public void displayBalance() {
+        System.out.println("Current balance:");
```
Опять же, раз мы работаем со спрингом, хотелось бы видеть нормальные логи а не System.out.print

BychkovVA committed:
Исправление замечаний
+ изменен порт
+ добавлено логирование
+ страницы вынесены в константы
+ АТМ - преобразован в компонент
